itest/test_harness.go

```diff
@@ -200,14 +202,19 @@ func (h *harnessTest) shutdown(_ *testing.T) error {
 func (h *harnessTest) setupLogging() {
 	h.logWriter = build.NewRotatingLogWriter()
 
+	logConfig := build.DefaultLogConfig()
+	h.logMgr = build.NewSubLoggerManager(
+		build.NewDefaultLogHandlers(logConfig, h.logWriter)...,
+	)
+
```

Comment:
i think same thing here as previous comment - i think we just want to print to console here so dont need to setup the rotating log writer. The binaries actually used in the itest will have their own rotating log writers which will write to a file but i think normal itest logs we just write to stdout usuall?